src/main/scala/org/labrad/registry/Registry.scala

```diff
@@ -69,6 +68,7 @@ object Registry {
 }
 
 class Registry(val id: Long, val name: String, store: RegistryStore, externalConfig: ServerConfig)
+              (implicit ec: ExecutionContext)
```

Comment:
I don't really like hanging indents because the indentation has to change if you change names, so names changes have a "blast radius" just to get the formatting right (and it's easy to forget). I think case I kinda threw my hands up after fiddling with a few options. The thing I don't like about the suggestion in the link is that then the `extends` clauses are indented at the same level as the class body, so it's hard to distinguish them. I really don't know what's best. I think the underlying issue is that the _grammar_ of the language is not very scalable when you have lots of class params, traits, etc.